.github/workflows/deploy-to-chromatic.yml

```diff
@@ -34,3 +31,51 @@ jobs:
           exitZeroOnChanges: true
           exitOnceUploaded: true
           autoAcceptChanges: 'main'
+  pr-deploy-to-chromatic:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/chromatic')
+    runs-on: ubuntu-latest
+    steps:
+      - if: github.event.issue.pull_request
+        name: Get PR branch
+        uses: alessbell/pull-request-comment-branch@v1.1
+        id: comment-branch
+      - uses: actions/checkout@v3.1.0
+        with:
+          repository: ${{ steps.comment-branch.outputs.head_owner }}/${{ steps.comment-branch.outputs.head_repo }}
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+          # Chromatic requires full git history to track changes
+          fetch-depth: 0
+      - uses: ./.github/actions/pnpm-setup-action
+      - name: Prebuild dependencies
+        # we technically only need to build the dependencies of SRC, but Nx doesn't expose an easy way to do that
+        run: pnpm nx run synapse-react-client:build
+      - id: chromatic-deploy
+        name: Publish to Chromatic
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          workingDir: packages/synapse-react-client
+          exitZeroOnChanges: true
+          exitOnceUploaded: false # Wait for snapshot tests to complete so we can report the results
+          autoAcceptChanges: 'main'
+          repositorySlug: ${{ steps.comment-branch.outputs.head_owner }}/${{ steps.comment-branch.outputs.head_repo }}
+          branchName: ${{ steps.comment-branch.outputs.head_ref }}
+      - name: Add comment to PR
```

Comment:
Like the idea of an automatically generated comment to surface these results in the PR

Reply:
I'd rather use the built-in status check on the commit (as we have now), but I can't seem to get that to work--looking at the Chromatic GH Action source code, I think they override any environment variables I might be able to pass. So this will have to do for now.

Reply:
If we use labels instead of comments, then we can continue to use the `pull_request_target` event, which Chromatic properly supports.